official/resnet/resnet_run_loop.py

```diff
@@ -355,21 +370,35 @@ def resnet_main(flags, model_function, input_function, shape=None):
       allow_soft_placement=True)
```

Comment:
The allow_soft_placement was only relevant for replicate_model_fn; @guptapriya -- still necessary to set this explicitly? I doubt it?

Reply:
Tagging @isaprykin on this one too-- do we need allow_soft_placement still?

Reply:
Sigh, yes, I think so.   Remember, this got added because Estimator wants allow_soft_placement=true and it sets it when session_config is not specified.  But when the session config is specified, then Estimator doesn't add allow_soft_placement.  Without allow_soft_placement, there device placement failures with no GPU kernels for some Ops...       I just looked at the Estimator code and I think this behavior is still in place.